### DISCUSSION TITLE
Octernship submission

### DISCUSSION THREAD

**Discussion by saaxena** [community/community#51664] [GitHub Education] [answered]
Hey folks ,  i want to know that where i have to submit by assignment . I had created a new branch and open a pull request so , should i merge it to main .
Or , we have to fork the project repo and then open a PR for making changes.

Reply asap

**Comment by ramazancetinkaya** [comment, marked as answer]
@saaxena, The answer to your question depends on the specific instructions provided by the project owner or instructor. However, in general, if you have created a new branch and made changes to the code, you should open a pull request (PR) to merge your changes into the main branch.

You should not merge your changes directly into the main branch unless you have been instructed to do so. Merging changes without review can lead to conflicts and errors in the codebase.

If you have been instructed to fork the project repository, you should create a fork of the repository, clone it to your local machine, make changes, push the changes to your forked repository, and then open a PR from your forked repository to the original repository.

In any case, it's always a good idea to read and follow the instructions provided by the project owner or instructor, and to communicate with them if you have any questions or concerns.

--- 

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Have a good day.

**Reply by saaxena** [reply]
Thanks for helping .

**Reply by shubhexists** [reply]
Hi guys, 
I may need a little more help regarding this please.
I have submitted my assignment to a repo I was added to by Github Classroom which was created the the company itself i.e. RustDesk.
May I know that anything further needs to be done or not?
Or maybe just help me go through the process you submitted yours please..
Thanks
